src/Neo/ProtocolSettings.cs

```diff
@@ -114,8 +114,8 @@ public record ProtocolSettings
             StandbyCommittee = Array.Empty<ECPoint>(),
             ValidatorsCount = 0,
             SeedList = Array.Empty<string>(),
-            MillisecondsPerBlock = 15000,
-            MaxTransactionsPerBlock = 512,
+            MillisecondsPerBlock = 3000,
+            MaxTransactionsPerBlock = 256,
```

Comment:
Why changing the default here? Why changing it at all? It doesn't matter how many transactions we have, execution resources are controlled with GAS.

Reply:
This is from vitor's previous PR, lets decides it during the meeting.

Reply:
May I ask what is `the time lock problem`? the last time i heard time lock is from hash time lock contract on lighting network.

Reply:
Now latest version only keeps the change to the block time.

Reply:
> May I ask what is the time lock problem?

Consider a contract that wants to lock some assets for some period of time, like a year. Developers know that blocks are ~15s and blocks are very convenient to count, so they can easily put a lock to expire at "current block + 2102400". And it's all fine except we're breaking this basic assumption of 15s and suddenly this lock expires somewhat earlier. Whether it's a problem or not for any real N3 mainnet contract is unknown, but it can be.

Reply:
Agree, it was recommended to use block number instead of time because time can be controlled by CN, now of we change this rule, contracts can be affected, we should ensure how many contracts use time o block height syscalls